cpmonitor/templates/index.html

```diff
@@ -29,6 +30,20 @@ <h2 class="card-title">Klimaneutrale Städte und Kommunen</h2>
       </div>
     </div>
   </div>
+  <div class="row row-deck row-cards">
+    {% for city in cities %}
+      <div class="col-sm-6 col-lg-3">
+        <a class="card" href="{% url 'city' city.slug %}">
+          <div class="card-body">
+            <div class="d-flex align-items-center">
+              <div class="subheader">{{ city.name }}</div>
```

Comment:
I think the city name is too small. It should not be a subheader but a header like on the task cards.